ext/json/json_parser.y

```diff
@@ -255,10 +255,40 @@ static int php_json_parser_object_update(php_json_parser *parser, zval *object,
 	return SUCCESS;
 }
 
+static int php_json_parser_array_create_validate(php_json_parser *parser, zval *array)
+{
+	ZVAL_EMPTY_ARRAY(array);
```

Comment:
Might be better to just do `ZVAL_NULL` here

Reply:
:heavy_check_mark: